src/ProtocolGateway.Core/Instrumentation/IPerformanceCounterManager.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.ProtocolGateway.Instrumentation
+{
+    public interface IPerformanceCounterManager
+    {
+        SafePerformanceCounter GetCounter(string category, string name);
```

Comment:
Same as above: In netstandard is used just to wrap EmptyPerformanceCounter so that PerformanceCounters class has the same properties as before.